tests/integrationv2/test_client_authentication.py

```diff
@@ -231,3 +240,53 @@ def test_client_auth_with_s2n_client_with_cert(managed_process, provider, other_
         assert b'read client certificate' in results.stderr
         assert b'read certificate verify' in results.stderr
         assert_openssl_handshake_complete(results)
+
+
+"""
+A s2n server that requests for a client certificate, and doesn't support RSA PSS,
+should downgrade to TLS 1.2 in case the client attempts to send an RSA PSS Certificate.     
+"""
+
```

Comment:
nit: you have some extra newlines here

Reply:
still some newlines here.

Reply:
You still have extra newlines here.

Reply:
Ah this must be from the auto-format stuff we did for python. Okay that's fine, we can keep it consistent.